pyproject.toml

```diff
@@ -110,7 +110,7 @@ exclude_lines = [
 ]
 
 [tool.pytest.ini_options]
-addopts = "--import-mode=importlib -k 'not veryslow'"
+addopts = "--import-mode=importlib"
```

Comment:
I don't think this should be the default here - this means that when people run `pytest vayesta` the `veryslow` tests will run, and it takes ages. I think this should just be controlled in the CI if we want them to always run, or just in the nightly scheduled run?